moto/kms/responses.py

```diff
@@ -278,9 +279,11 @@ def list_aliases(self):
                 a for a in response_aliases if a["AliasName"] == reserved_alias
             ]
             if not exsisting:
+                arn = f"arn:aws:kms:{region}:{self.current_account}:{reserved_alias}"
                 response_aliases.append(
                     {
-                        "AliasArn": f"arn:aws:kms:{region}:{self.current_account}:{reserved_alias}",
+                        "TargetKeyId": str(uuid.UUID(int=abs(hash(arn)), version=4)),
```

Comment:
The `hash`-function is only guaranteed to give the same result within the same Python runtime. 
That probably explains the test failure - the hash as calculated during the test is in a different runtime than the Moto server, and therefore yields a different result.

Does it make sense to use a random UUID instead? Having a reproducible value is nice for our tests, but this is not guaranteed in AWS as far as I know.

Reply:
True story. Admittedly I addressed my thought in an overly complicated manner now that I'm looking back at it.

They will be different across accounts for sure, but what I wanted to avoid was getting a random uuid back per request. Within one account, the same id's ought to be returned.

I will go look at it and see if I can't come up with a simpler approach!